xtend-website/documentation/203_xtend_expressions.md

```diff
@@ -919,6 +932,17 @@ val name = try {
   }
 ```
 
+Handling multiple exceptions in a single catch block can be realized using a *multi-catch clause*. The *multi-catch clause* can handle an arbitrary number of exceptions separated by `|`. Within the catch block, the intersection exception type of the exceptions caught by the *multi-catch clause* is available for further processing.
```

Comment:
Please add a note that error is raised when one exception covers another, e.g. `NullPointerException | RuntimeException`